test/basic.js

```diff
@@ -57,12 +84,7 @@ tape('encode/decode bendy butt', function (t) {
   const encoded = bfe.encodeBendyButt(values)
   t.equal(encoded[0].slice(0, 2).toString('hex'), '0003', 'bendy feed')
   t.equal(encoded[1].slice(0, 2).toString('hex'), '0104', 'bendy msg')
-  const nullBuffer = Buffer.concat([
-    Buffer.from([1]),
-    Buffer.from([4]),
-    Buffer.alloc(32),
-  ])
-  t.equal(encoded[2].compare(nullBuffer, 0, 32 + 2), 0, 'bendy null msg')
+  t.equal(encoded[2].toString('hex'), '0602', 'null')
```

Comment:
I think these should be encoded as 010400000000000000000000000000000000?

Reply:
Well, the whole point of https://github.com/ssb-ngi-pointer/ssb-binary-field-encodings-spec/issues/4#issuecomment-879746796 is to encode these as `0602` :sweat_smile:

Reply:
hmm, but that won't work with box2. It would a lot easier if we could do this I totally agree, but I don't think we are able to change this now.

Reply:
Did you see https://github.com/ssb-ngi-pointer/ssb-binary-field-encodings-spec/issues/4#issuecomment-879793980 ? Is there anything missing there that box2 requires?